src/coreclr/scripts/superpmi.py

```diff
@@ -1608,6 +1608,13 @@ def replay_with_asm_diffs(self):
                 if self.coreclr_args.error_limit is not None:
                     flags += ["-failureLimit", self.coreclr_args.error_limit]
 
+                if self.coreclr_args.diff_with_release:
+                    # If one of the JITs is Release, ignore all the stored configuration variables.
+                    # This isn't necessary if both are Release builds (and, in fact, it can clear variables
+                    # that both Release compilers can interpret). However, we rarely or never compare
+                    # two Release compilers, so this is safest.
```

Comment:
I was thinking of moving asmdiffs to run on Release instead of Checked.

Reply:
When doing so, you'll need to audit usage of `diff_with_release`. It was introduced for use with Checked/Release diffs (so is not completely accurately named) and also before Release builds had any JitDisasm support (JitDiffableDasm still isn't in Release), so I think it kind of mixes concepts a bit.